apps/ui/src/composables/useAccount.ts

```diff
@@ -32,7 +35,7 @@ export function useAccount() {
     const network = getNetwork(networkId);
     const userVotes = await network.api.loadUserVotes(spaceIds, account);
 
-    votes.value = { ...votes.value, ...userVotes };
+    votes.value = userVotes;
```

Comment:
How does it affect [this](https://github.com/snapshot-labs/sx-monorepo/pull/518#discussion_r1687597637)?

Reply:
Reverted, this fix was extracted and improved in #518